apps/src/lib/client/rpc.rs

```diff
@@ -1581,12 +1581,17 @@ pub async fn query_bonded_stake<C: namada::ledger::queries::Client + Sync>(
                         .await,
                 );
 
+            let sorted_consensus = consensus.into_iter().sorted_by_key(|v| {
```

Comment:
I think we could probably just return `BTreeSet` from the set queries as the `WeightedValidator` already has order

Reply:
done in b3b1f261049f03cf1a70da7f378e806c47204500